tensorflow/compiler/mlir/tosa/transforms/strip_quant_types.cc

```diff
@@ -52,69 +55,28 @@ namespace {
 #define GEN_PASS_CLASSES
 #include "tensorflow/compiler/mlir/tosa/transforms/passes.h.inc"
 
-class StripQuantTypes : public TosaStripQuantTypesPassBase<StripQuantTypes> {
- public:
-  explicit StripQuantTypes() {}
-  void runOnFunction() override;
-};
-
-class QuantTypeConverter : public TypeConverter {
```

Comment:
Type converters are specifically built to handling multiple type conversion and should not be removed. This violates how MLIR tries to handle type conversion. Even if dialect conversion is not being used, a TypeConverter should be maintained.